test/stringio/test_stringio.rb

```diff
@@ -798,7 +798,7 @@ def test_new_block_warning
   end
 
   def test_overflow
-    skip if RbConfig::SIZEOF["void*"] > RbConfig::SIZEOF["long"]
+    pend if RbConfig::SIZEOF["void*"] > RbConfig::SIZEOF["long"]
```

Comment:
Could you use `omit` not `pend` for `skip`? `pend` is for `pending` in RSpec.

```suggestion
    omit if RbConfig::SIZEOF["void*"] > RbConfig::SIZEOF["long"]
```